custom_components/zwave_mqtt/__init__.py

```diff
@@ -101,7 +102,7 @@ def async_node_removed(node):
         # entities itself are removed by the values logic
         if node.id in removed_nodes:
             hass.async_create_task(handle_remove_node(hass, node.id))
-            removed_nodes.remove(node.id)
+            removed_nodes.remove(node)
```

Comment:
This doesn't look correct.

Reply:
That's what you get when you are too quick and that's why we have reviews ;-)
Thanks